engine/seq/start.py

```diff
@@ -22,6 +26,19 @@ def __init__(self, saveslot: int):
                                     name="SYSTEM",
                                     text="Press confirm to select new game.",
                                 ),
+                                SeqOptional(
```

Comment:
Note: Made this sequence more generic, so it can now start both Evoland 1 and Evoland 2 from the main menu. The load game procedure is the same for both games, and doesn't require any changes here.